ci/build-msys2.sh

```diff
@@ -47,11 +47,10 @@ if [ "$1" = "waf" ]; then
       --enable-rubberband      \
       --enable-shaderc         \
       --enable-spirv-cross     \
-      --enable-tests           \
       --enable-uchardet        \
       --enable-vapoursynth     \
       --lua=luajit             \
       --enable-vulkan
     ./waf build
-    ./build_waf/mpv.com -v --no-config -v --unittest=all-simple
+    ./build_waf/mpv.com -v --no-config
```

Comment:
Could sync this command between all build scripts. Some do `-v --no-config` other don't.